packages/ecs-agent/0001-bottlerocket-default-filesystem-locations.patch

```diff
@@ -41,37 +30,11 @@ index 38f0d0f..acebb26 100644
  
  	// DefaultTaskMetadataSteadyStateRate is set as 40. This is arrived from our benchmarking
  	// results where task endpoint can handle 4000 rps effectively. Here, 100 containers
-diff --git a/agent/config/config_unix.go b/agent/config/config_unix.go
-index 6c1254d..3adefaa 100644
---- a/agent/config/config_unix.go
-+++ b/agent/config/config_unix.go
-@@ -29,10 +29,10 @@ const (
- 	// AgentCredentialsAddress is used to serve the credentials for tasks.
- 	AgentCredentialsAddress = "" // this is left blank right now for net=bridge
- 	// defaultAuditLogFile specifies the default audit log filename
--	defaultCredentialsAuditLogFile = "/log/audit.log"
-+	defaultCredentialsAuditLogFile = "/var/log/ecs/audit.log"
- 
- 	// defaultRuntimeStatsLogFile stores the path where the golang runtime stats are periodically logged
--	defaultRuntimeStatsLogFile = `/log/agent-runtime-stats.log`
-+	defaultRuntimeStatsLogFile = `/var/log/ecs/agent-runtime-stats.log`
```

Comment:
Does this not need to be set any longer? I didn't see `agent-runtime-stats.log` appear in one of the config places.

Reply:
We are not setting it anymore. The config to enable them is set to false by default and we don't expose any api to change that config. @arnaldo2792 suggested this as per https://github.com/aws/amazon-ecs-agent/blob/762d184f3ab45c239f9f208febb9d3c362053e9f/agent/config/config_unix.go#L108